packages/@okta/vuepress-site/docs/guides/dpop/main/oktaresourceserver/claims.md

```diff
@@ -0,0 +1,3 @@
+* `htm`: HTTP method. The HTTP method of the request that the JWT is attached to. This value is always `POST`.
+* `htu`: HTTP URI. The `/token` endpoint URL for the Okta authorization server that you want to use. Example: `http://${yourOktaDomain}/oauth2/v1/token`.
+* `iat`: Issued at. Identifies the time at which the JWT is issued. The time appears in seconds since the Unix epoch. The Unix epoch is the number of seconds that have elapsed since January 1, 1970 at midnight UTC.
```

Comment:
```suggestion
* `iat`: Issued at. The time at which the JWT is issued. The time appears in seconds since the Unix epoch. The Unix epoch is the number of seconds that have elapsed since January 1, 1970 at midnight UTC.
```

Reply:
For parallelism with other list items (no verbs). This appears to be the same content as the claims.md (snippet?) above. Both needed?[/dpop/main/nonoktaresourceserver/claims.md]

Reply:
Yes - this is the new content. The files in the "nonoktaresourceserver" folder is existing, live, and approved. I updated that file per your edit. Altho `iat` doesn't have a change - the `htu` item in this list does.